worldengine/cli/main.py

```diff
@@ -482,6 +500,26 @@ def main():
         print(' draw rivers            : %s' % args.draw_rivers)
         print(' draw mountains         : %s' % args.draw_mountains)
         print(' draw land outer border : %s' % args.draw_outer_border)
+        
+    #Warning messages
+    warnings = []
+    if temps != sorted(temps, reverse=True):
+        warnings.append("WARNING: Temperature array not in ascending order")
+    if numpy.amin(temps) < 0:
+        warnings.append("WARNING: Maximum value in temperature array greater than 1")
```

Comment:
No, the warnings are actually correct. We invert the values that are given because I think it will be easier for people to understand and I worry that if I go through all of the code to handle the new values directly I'll end up missing bits and it will end up not working correctly.